src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        ColorSupplier supplier = new ColorSupplier();
+        for (int i = 0; i < 3; i++) {
```

Comment:
To avoid creating multiple instances of `ColorSupplier` and `Lottery` inside the loop, you should instantiate them once before the loop starts and then use these instances within the loop.